### DISCUSSION TITLE
Problem with 'temporarily inactive' label

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#510] [Q&A] [answered]
I have a phone meeting, that I cannot process in the plugin without it showing the 'temporarily inactive' label.
Does anyone have a solution? There is no phone number (people have to reach out to an email address to get it), there is no URL. There is just a note that it is phone only. Suggestions?

Thanks,
Nicky

**Comment by tech2serve** [comment, marked as answer]
Hi Nicky--

There is another thread on this you can find and review for more context, but the gist is that certain conditions must be met if a meeting is going to show up in TSML.

1. For online, it *must* have a valid conference URL or phone number. 
2. For in-person, it *must* have a specific, non-approximate, address to which users can map.
3. For hybrid, it *must* have both.

Meeting entries not satisfying those conditions will show as inactive, or not publish.

The decision was made long ago that a primary purpose of TSML is to help users get to meetings by providing them all the information they need to attend a particular meeting.

Consequently, the plugin doesn't support call-to-get directions-meetings.

So for an phone meeting, I suggest configuring as follows:

1. Include a valid phone number for the meeting. If you're adamant about not publicly listing that number, then you can include a central office/Intergroup number for users to call.
2. Assuming the meeting is not hybrid, make sure the address is approximate (i.e., _only_ enter city and state).

If you do these steps, the meeting *should* publish and show as an online meeting.

If this is not the solution the group wants, then we suggest removing the meeting from the database and listing it separately on your website so users can find it there.

**Comment by OSAwebsite** [comment]
It would be helpful to get a link to that other thread. I looked for it last night but could not find it. Maybe I am just suffering from a foggy brain, but would you mind guiding me to that thread by title or link?

**Reply by tech2serve** [reply]
> It would be helpful to get a link to that other thread. I looked for it last night but could not find it. Maybe I am just suffering from a foggy brain, but would you mind guiding me to that thread by title or link?

Recent discussion on "Meeting showing as inactive when it shouldn't do" is one example (this was a thread hijack). There are more back in the Issues.

**Comment by OSAwebsite** [comment]
OK, thanks.
